src/routes/Game.vue

```diff
@@ -87,17 +88,28 @@ export default {
 
         this.instance = new GameInstance(this);
 
+        if(this.$route.params.sheet && this.$route.params.sheet!="null"){
```

Comment:
This is comparing `this.$route.params.sheet` with a "string" literal `"null"`. Is this intended?

Reply:
Well it is not... But only as a temporary solution, checks for id and URL existence should be in place here in the future.

Reply:
Then I suggest adding a `FIXME` comment here for future reference ❤️